### ISSUE TITLE
UnicodeDecodeError: 'ascii' codec can't decode byte

### ISSUE THREAD

**Issue by tenzion** [nzbget/VideoSort#16]
Running latest version together with nzbget v18.0.22 on my Synology.
Is it possible to disable "Lower words" and "Upper words" altogether?

Thank you.

**Comment by hugbug** [comment]
Have you tried setting options "LowerWords" and "UpperCase" to empty values?

**Comment by tenzion** [comment]
Yes. It did not do any difference.
Problem is, this is a movie in a non-english language, and I can't put every single word into the list so that VideoSort can avoid a particular word or title...

**Comment by hugbug** [comment]
I don't understand. Please provide an example of movie title and the result you get in VideoSort and the result you expect.

**Comment by tenzion** [comment]
An example could be:
This is what I have: "Far til fire gi'r aldrig op (2005)"
and this is the result: "Far Til Fire Gi'R Aldrig Op (2005)"

It moves the file and folder alright, that works. But renaming does not.

**Comment by hugbug** [comment]
> and the result you expect.

This is missing in your post.

**Comment by tenzion** [comment]
Eh. 
Let me try again:
This is what I have: "Far til fire gi'r aldrig op (2005)"
and this is the result: "Far Til Fire Gi'R Aldrig Op (2005)"
All starting letters after the first "Far" have now been changed to upper letters.

**Comment by hugbug** [comment]
And you still don't say what you want.
Assumung you want the letter case preserved as in original nzb name, you should use "%tT" instead of "%t". See description of formatting specifier.

**Comment by tenzion** [comment]
Doh.
Sorry for not reading through the whole list with % options...
The first hit on the list was good enough for me apparently :-)
I see it now! 
Thanks for the help!

**Comment by tenzion** [comment]
Btw, just another question:
O	Fri Apr 21 2017 13:59:19	VideoSort: filename: /volume1/downloads/complete/movies/Far til fire_ Til søs (2012) - 1080p - DTS/Far til fire - Til soes (2012) 1080p - DTS.mkv
INFO	Fri Apr 21 2017 13:59:19	VideoSort: UnicodeDecodeError: 'ascii' codec can't decode byte 0xc3 in position 19: ordinal not in range(128)
INFO	Fri Apr 21 2017 13:59:19	VideoSort: matcher = guessit.matcher.IterativeMatcher(unicode(guessfilename), filetype='autodetect', options={'nolanguage': True, 'nocountry': True})
What could this indicate in the renaming process?

**Comment by hugbug** [comment]
Please see #3. In the last post a fix is suggested. If you can try it and report back that would be great.

**Comment by hugbug** [comment]
Closing this issue. For further discussion of UnicodeDecodeError please use #3.
